libs/ui/src/themes/render_with_themes.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render, RenderOptions, RenderResult } from '@testing-library/react';
+
+import { ColorMode, SizeMode } from '@votingworks/types';
+import { AppBase } from '../app_base';
+
+type VxRenderOptions = RenderOptions & {
+  vxTheme?: {
+    colorMode?: ColorMode;
+    sizeMode?: SizeMode;
+  };
+};
+
+/**
+ * React testing render function with UI theme support.
+ * This is needed when rendering component trees that contain theme-dependent
+ * components from libs/ui.
+ */
+export function renderWithThemes(
+  ui: React.ReactElement,
+  options: VxRenderOptions = {}
+): RenderResult {
+  const { vxTheme = {}, ...passthroughOptions } = options;
+
+  function wrapper(props: { children: React.ReactNode }): JSX.Element {
+    return (
+      <AppBase
+        colorMode={vxTheme.colorMode}
```

Comment:
Just added support for specifying theme params while rendering to make life a little easier in future PRs